test/standalone/test_rpc_proxy.py

```diff
@@ -726,7 +796,25 @@ def enable_after_retry(args, kwargs, res, exc_info):
 
 
 @skip_if_no_toxiproxy
-class TestStandaloneProxyConsumerDisconnections(object):
+class TestStandaloneProxyReplyListenerDisconnections(object):
```

Comment:
The diff is messy here. `TestStandaloneProxyReplyListenerDisconnections` is completely new and adds the standard disconnection tests now that the standalone reply listener follows the standard "consumer" pattern.